frontend/src/components/Item/ItemMenu.vue

```diff
@@ -57,6 +54,7 @@ import IMdiPencilOutline from 'virtual:icons/mdi/pencil-outline';
 import IMdiShuffle from 'virtual:icons/mdi/shuffle';
 import IMdiReplay from 'virtual:icons/mdi/replay';
 import IMdiRefresh from 'virtual:icons/mdi/refresh';
+import mdiDotsHorizontal from 'virtual:icons/mdi/dots-horizontal';
```

Comment:
```suggestion
import IMdiDotsHorizontal from 'virtual:icons/mdi/dots-horizontal';
```

**PascalCase**. Told you in Matrix, but leaving this here as a note as well. Do the same for all your icon imports please (also, maybe we can investigate if we can enforce this through ``eslint-plugin-imports``

Reply:
Done